openapiart/bundler.py

```diff
@@ -884,6 +956,7 @@ def _generate_value_schema(
                     "offset": {
                         "description": "Offset in bits relative to start of corresponding header field",
                         "type": "integer",
+                        "format": "uint32",
```

Comment:
why is this hardcoded ? shouldn't it depend on bit length ?

Reply:
done